### PR TITLE
pkg: fix fragments should contain more then one child

### PR DESCRIPTION
**Pull request by jelly**
`oxlint` a Rust implementation of `eslint` finds more issues for the `jsx-no-useless-fragment` eslint react rule. The Rust implementation is not feature complete yet so we can't yet switch over.

https://oxc-project.github.io/blog/2023-12-12-announcing-oxlint.html

### CODE REVIEW

File: pkg/networkmanager/dialogs-common.jsx [hunk 1]
```diff
@@ -254,14 +254,12 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
     }
 
     return (
-        <>
-            <Button id={"networking-" + (!iface ? "add-" : "edit-") + type}
+        <Button id={"networking-" + (!iface ? "add-" : "edit-") + type}
                     isInline={!!iface}
                     onClick={syn_click(model, show)}
                     variant={!iface ? "secondary" : "link"}>
```
Comment (martinpitt) [inline pkg/networkmanager/dialogs-common.jsx:260]: Please fix the indentation.

File: pkg/packagekit/updates.jsx [hunk 2]
```diff
@@ -1587,24 +1586,22 @@ class OsUpdates extends React.Component {
...
+                    <Gallery className='ct-cards-grid' hasGutter>
+                        <CardsPage onValueChanged={this.onValueChanged} handleRefresh={this.handleRefresh} {...this.state} />
+                    </Gallery>
+                    { this.state.showRestartServicesDialog &&
+                    <RestartServices tracerPackages={this.state.tracerPackages}
                                 close={() => this.setState({ showRestartServicesDialog: false })}
                                 state={this.state.state}
                                 callTracer={(state) => this.callTracer(state)}
                                 onValueChanged={delta => this.setState(delta)}
                                 loadUpdates={this.loadUpdates} />
```
Comment (martinpitt) [inline pkg/packagekit/updates.jsx:1599]: Same here -- either the properties should align (prefered), or it should at least have a consistent 4-space indentation).

**Review by martinpitt — CHANGES_REQUESTED** [review]
Hah, nice cleanup!

**Review by martinpitt — APPROVED** [review]
Cheers!

File: pkg/networkmanager/dialogs-common.jsx (hunk 1 above)
Comment (jelly) [reply pkg/networkmanager/dialogs-common.jsx]: Seems this comes up too much :) I think we should really let eslint handle this better but that requires some starter-kit / podman

**Pull request merged**